cli.js

```diff
@@ -8,7 +8,7 @@ var debug = require('debug')('ava');
 var resolveCwd = require('resolve-cwd');
 var localCLI = resolveCwd('ava/cli');
 
-if (localCLI && localCLI !== __filename) {
+if (localCLI && path.relative(localCLI, __filename) !== '') {
```

Comment:
Can you add a comment referencing https://github.com/nodejs/node/issues/6624 for posterity. Otherwise someone will come by and try to simplify the code without understanding what it's solving.